### PR TITLE
adds citation and manafest files

### PR DESCRIPTION
**Pull request by alan-stokes**
adds citation and manifest files. start point for further work on creating a doi for the tools

### CODE REVIEW

**Comment by dkfellows** [comment]
Question: I can't see any substantive contributions to this repository by @alan-stokes. All (accepted-to-`master`) contributions appear to be merge commits. Should he be listed as an author?

**Comment by andrewgait** [comment]
Personally speaking - yes, as by reviewing and doing merge commits he is contributing... ? It's almost analogous to the situation of someone contributing via ideas rather than by actual code.

To put it another way, if there was something wrong with the merge commit, then `git blame` would blame him, wouldn't it?

**Comment by rowleya** [comment]
I don't think it would - I think merge commits are excluded.   Code review would usually just be considered the equivalent to proof reading, but it depends on if what was changed in response I guess.  The main thing to think about is whether the author can understand the code enough to be able to make changes happily, or whether this would currently require them to learn the code significantly...  I wouldn't want to be author on a paper that I didn't really understand as I wouldn't want to be asked questions about it!

File: pypi_to_import [hunk 1]
```diff
@@ -0,0 +1,5 @@
+six:six
```
Comment (dkfellows) [inline pypi_to_import]: Can you make your tool assume that the two names match each other unless told otherwise? That'd keep these files shorter.

**Comment by alan-stokes** [comment]
how about i just remove myself from this repo's citation file. like i said to donal personally. A bit of this is copy and paste, and following contributors on github. Im glad he's chekcing the sanity of it all.

**Comment by alan-stokes** [comment]
done.

File: pypi_to_import (hunk 1 above)
Comment (alan-stokes) [reply pypi_to_import]: i persume so. will look into it once ive got the base logic sound

**Pull request merged**